interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +127,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$g_edit && $have_group_global_enabled )?' disabled=true; ':'';?>
+<?php if($disabled) echo '<script>$( document ).ready(function(){
+    $("input").prop("disabled", true); 
+    $("select").prop("disabled", true); 
```

Comment:
Will this disable all input and select elements on the form if this is a standard appointment (ie. not a group appt)?